backend/src/graphql/resolvers/TableResolver.spec.ts

```diff
@@ -317,6 +317,35 @@ describe('TableResolver', () => {
             },
           })
         })
+
+        it('creates create my table event in the database', async () => {
```

Comment:
```suggestion
        it('logs event `CREATE_MY_TABLE`', async () => {
```

We could also specify the payload here.

Reply:
This is not meant as a log. The purpose is, that admins can see the activity of certain users, e.g. to provide the support with information concerning the user asking for help. Furthermore, you can easily generate statistics of the usage of the application. Doing this by a database log is much harder to achieve.

Reply:
Does this answer your question?

Reply:
Alright, I've hit approve. Fine with me to save it to the database if "admins" are non-technical admins. But we have no UI for this yet, right?